### PR TITLE
3022: Ajax load user menu to prevent caching in varnish

### PR DESCRIPTION
**Pull request by cableman**
See https://platform.dandigbib.org/issues/3022

### CODE REVIEW

File: themes/ddbasic/sass/components/menu.scss [hunk 1]
```diff
@@ -559,3 +559,43 @@ ul.topbar-menu {
     }
   }
 }
+
+// ==========================================================================
+// User mobile menu placeholder - lazy loading
+// ==========================================================================
+.mobile-user-menu .lazy-pane-placeholder {
```
Comment (kasperg) [inline themes/ddbasic/sass/components/menu.scss]: This seems to set a new spinner specifically for this element. Is that really necessary?

File: themes/ddbasic/sass/components/menu.scss [hunk 2]
```diff
@@ -559,3 +559,43 @@ ul.topbar-menu {
...
+    left: 0;
+    width: 100%;
+    height: 4px;
+    content: "";
+  }
+
+  .mobile-usermenu-is-open & {
+    display: block;
+  }
+}
```
Comment (kasperg) [inline themes/ddbasic/sass/components/menu.scss]: Missing newline.

File: themes/ddbasic/templates/menu/panels-pane--sub-menu--mobile.tpl.php [hunk 3]
```diff
@@ -20,11 +20,12 @@
...
-</section>
+<?php if ($is_lazy_pane_render) { ?>
+  <?php print render($content); ?>
+<?php } else { ?>
+  <section class="navigation-wrapper mobile-user-menu <?php print $classes; ?>">
+    <div class="navigation-inner">
+      <?php print render($content); ?>
+    </div>
+  </section>
+<?php } ?>
```
Comment (kasperg) [inline themes/ddbasic/templates/menu/panels-pane--sub-menu--mobile.tpl.php]: This seems to remove `$admin_links` entirely from the renderend template. Is that intentional? I could not tell from the commit comment.

File: themes/ddbasic/sass/components/menu.scss (hunk 1 above)
Comment (cableman) [reply themes/ddbasic/sass/components/menu.scss]: It's due to the way these spinners is constructed. I did not see any other way to get the spinner here without having to refactor the SASS. Which would be outside the scope of this issue.

File: themes/ddbasic/sass/components/menu.scss (hunk 2 above)
Comment (cableman) [reply themes/ddbasic/sass/components/menu.scss]: Fixed

File: themes/ddbasic/templates/menu/panels-pane--sub-menu--mobile.tpl.php (hunk 3 above)
Comment (cableman) [reply themes/ddbasic/templates/menu/panels-pane--sub-menu--mobile.tpl.php]: I don't think they would every be set here, but re-add theme to the template to be sure.

**Pull request merged**